wweval/R/ms_fig4.R

```diff
@@ -484,13 +485,36 @@ make_fig4_rel_crps_by_phase <- function(scores) {
     geom_hline(aes(yintercept = 1), linetype = "dashed") +
     xlab("Epidemic phase") +
     ylab("Relative CRPS") +
-    scale_y_continuous(trans = "log10") +
+    scale_y_continuous(trans = "log10", limits = c(0.5, 2)) +
     get_plot_theme(
       x_axis_title_size = 8,
       y_axis_title_size = 8
     ) +
     scale_fill_brewer(palette = "Set3")
 
+  # Going to keep this in there for now
+  scores_to_plot <- scores_w_fig_order |>
+    dplyr::filter(!is.na(phase))
+
+  ggplot(scores_to_plot) +
+    tidybayes::stat_halfeye(
+      aes(
+        x = as.factor(phase), y = crps,
+        fill = model
+      ),
+      point_interval = "mean_qi",
+      alpha = 0.5,
+      position = position_dodge(width = 0.75),
+    ) +
+    xlab("Epidemic phase") +
+    ylab("CRPS") +
+    scale_y_continuous(limits = c(0.0, 1)) +
+    get_plot_theme(
+      x_axis_title_size = 8,
+      y_axis_title_size = 8
+    ) +
+    scale_fill_manual(values = colors$model_colors)
```

Comment:
If you want to hold onto this code within this file, it should be moved into a separate function (which just never gets called).

Related: I'd consider modularizing this function into a `compute_relative_crps` function and a `plot_relative_crps` function

Reply:
First point should be addressed in this PR, second doesn't have to be.

Reply:
Yeah thats a good point, I can do both in this PR.

Reply:
https://github.com/CDCgov/wastewater-informed-covid-forecasting/issues/131